test/widgets/autocomplete_test.dart

```diff
@@ -108,25 +109,21 @@ void main() {
 
   group('ComposeAutocomplete', () {
 
-    Finder findNetworkImage(String url) {
-      return find.byWidgetPredicate((widget) => switch(widget) {
-        Image(image: NetworkImage(url: var imageUrl)) when imageUrl == url
-          => true,
-        _ => false,
-      });
-    }
+    Finder findAvatarImage(int userId) =>
+      find.byWidgetPredicate((widget) => widget is AvatarImage && widget.userId == userId);
 
     void checkUserShown(User user, PerAccountStore store, {required bool expected}) {
       check(find.text(user.fullName).evaluate().length).equals(expected ? 1 : 0);
+      check(find.text(user.deliveryEmail!).evaluate().length).equals(expected ? 1 : 0);
       final avatarFinder =
-        findNetworkImage(store.tryResolveUrl(user.avatarUrl!).toString());
+        findAvatarImage(user.userId);
```

Comment:
This can now fit on one line, like in your previous revision:

```dart
final avatarFinder = findAvatarImage(user.userId);
```

Reply:
Okay. Let me quickly do that